src/lib/Microsoft.Fx.Portability.Reports.Html/HtmlRazorReportWriter.cs

```diff
@@ -71,12 +74,57 @@ private static ServiceProvider BuildProvider()
 
             services.AddSingleton<ObjectPoolProvider, DefaultObjectPoolProvider>();
             services.AddLogging();
-            services.AddMvc();
-            services.AddTransient<RazorViewToStringRenderer>();
+            services
+                .AddMvc()
+                .ConfigureApplicationPartManager(partsManager => ConfigureRazorViews(partsManager, fileProvider));
 
+            services.AddTransient<RazorViewToStringRenderer>();
             return services.BuildServiceProvider();
         }
 
+        /// <summary>
+        /// Checks if the precompiled razor views have been added as an
+        /// ApplicationPart. If not, adds them.
+        /// On .NET 461, Microsoft.Fx.Portability.Reports.Html.dll
+        /// and Microsoft.Fx.Portability.Reports.Html.Views.dll are
+        /// not added to the ApplicationPartsManager by default.
+        /// For more information: https://github.com/aspnet/Razor/issues/2262.
+        /// </summary>
+        private static void ConfigureRazorViews(ApplicationPartManager partsManager, PhysicalFileProvider fileProvider)
+        {
+            var comparison = StringComparison.OrdinalIgnoreCase;
```

Comment:
Is there a reason to do this? Why not just assume the naming is how we expect?

Reply:
When I was testing it on .NET FX, I noticed that the files were being output as .DLL rather than .dll, so it couldn't find the assembly.

Reply:
Can you add a comment to that effect?